src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,132 +18,359 @@ should be included in the pipeline documentation.
+| `shore`     | Simple Harmonic Oscillator-based Reconstruction and Estimation \[[Ozarslan2008](#ozarslan2008)\]                                                |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `wmti`      | White Matter Tract Integrity \[[Fieremans2011](#fieremans2011)\]                                                                                | One [parameter vectors](#parameter-vectors) image with parameter name "`coeffs`", with 33 volumes in the order: *D<sub>xx</sub>*, *D<sub>xy</sub>*, *D<sub>xz</sub>*, *D<sub>yy</sub>*, *D<sub>yz</sub>*, *D<sub>zz</sub>*, *W<sub>xxxx</sub>*, *W<sub>yyyy</sub>*, *W<sub>zzzz</sub>*, *W<sub>xxxy</sub>*, *W<sub>xxxz</sub>*, *W<sub>xyyy</sub>*, *W<sub>yyyz</sub>*, *W<sub>xzzz</sub>*, *W<sub>yzzz</sub>*, *W<sub>xxyy</sub>*, *W<sub>xxzz</sub>*, *W<sub>yyzz</sub>*, *W<sub>xxyz</sub>*, *W<sub>xyyz</sub>*, *W<sub>xyzz</sub>*, *Dh<sub>xx</sub>*, *Dh<sub>xy</sub>*, *Dh<sub>xz</sub>*, *Dh<sub>yy</sub>*, *Dh<sub>yz</sub>*, *Dh<sub>zz</sub>*, *Dr<sub>xx</sub>*, *Dr<sub>xy</sub>*, *Dr<sub>xz</sub>*, *Dr<sub>yy</sub>*, *Dr<sub>yz</sub>*, *Dr<sub>zz</sub>* (*D* is the diffusion tensor and *W* is the kurtosis tensor);<br>One [scalar](#scalar) image with parameter name "`awf`", representing the estimated axonal water fraction |
+
+The JSON sidecar for the intrinsic diffusion model parameters may contain
+the following key/value pairs irrespective of the particular model:
+
+| **Key name**     | **Description**                                                                                                                                                                              |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Gradients        | OPTIONAL. List of 3-vectors. Subset of gradients utilized to fit the model, as a list of three-elements lists. If not present, all gradients were used.                                      |

```

Comment:
```suggestion
| Gradients        | REQUIRED. List of 3-vectors. Subset of gradients utilized to fit the model, as a list of three-elements lists.                                                                               |
```

Reply:
If these two fields are not required we will easily end up having all models use all the gradients always.

Reply:
I would expect the majority of model implementations to be compatible with using all data, as well as all experimental uses of such to have utilised all data. So the necessity to clarify which of the input data were used should in fact be quite rare. What could be said instead is that the field is REQUIRED if the complete set of input DWI data were not used.

See also concern RE: `Gradients` and `Shells` fields in first PR comment.